tools/db_bench_tool.cc

```diff
@@ -8097,6 +8098,15 @@ int db_bench_tool(int argc, char** argv) {
   FLAGS_use_existing_db |= FLAGS_readonly;
 #endif  // ROCKSDB_LITE
 
+  if (!FLAGS_seed) {
+    uint64_t now = FLAGS_env->GetSystemClock()->NowMicros();
+    seed_base = static_cast<int64_t>(now);
```

Comment:
Is it going to change result of performance result: https://github.com/facebook/rocksdb/wiki/Performance-Benchmarks

Reply:
It can for benchmarks that have been using it incorrectly. And by incorrect I usually mean:
* buffered IO is used
* database is larger than memory
* db_bench run multiple times

In this case the first runs warm up the OS page cache, and the following tests can benefit from great cache hit rates. This is usually a mistake. Tests that want it can just do --seed=X where X is != 0 but constant for all db_bench runs.

tools/benchmark.sh and tools/regression_test.sh avoid this problem via --seed=$( date +%s )